specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/examples/DdosCustomPolicyCreate.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "api-version": "2018-10-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "ddosCustomPolicyName": "test-ddos-custom-policy",
+    "parameters": {
+      "location": "westus",
+      "properties": {}
```

Comment:
You need to provide atleast 1 protocol custom setting in the custom policy else the PUT is not allowed